scout/load/disease.py

```diff
@@ -38,20 +37,35 @@ def load_disease_terms(
         hpo_annotation_lines = fetch_hpo_disease_annotation()
     disease_annotations = parse_hpo_annotations(hpo_annotation_lines)
 
+    LOG.info(
+        f"Before adding to terms: length of annotations: {len(disease_annotations)} and length of disease_terms: {len(disease_terms)}"
+    )
+
+    # Update disease_terms with OMIM-terms from disease_annotations if not already present
+    for disease_id, content in disease_annotations.items():
+        if disease_id not in disease_terms and content["source"] == "OMIM":
+            disease_terms[disease_id] = {
+                "mim_number": content["disease_nr"],
+                "inheritance": set(),
+                "description": content["description"],
+                "status": None,
+                "hgnc_symbols": set(),
```

Comment:
```suggestion
                "hgnc_symbols": content["hgnc_symbols"],
```

Perhaps some gene might be available also in this file?

Reply:
Good point!